jukebox_radio/streams/models/queue_interval_model.py

```diff
@@ -146,7 +251,6 @@ class QueueInterval(models.Model):
     )
 
     is_muted = models.BooleanField()
-    repeat_count = models.PositiveSmallIntegerField(null=True)
```

Comment:
this isn't in the spec right now. too big of a task to add at this point. would nice to be add later.